pages/about.html

```diff
@@ -3,10 +3,95 @@
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel = "stylesheet" href = "../styles/about.css">
+  <title>About Me</title>
 </head>
 <body>
-  
+  <header>
+    <h1>About Me</h1>
+  <nav class = "navbar">
+    <a href = "index.html">HOME</a> 
+    <a href = portfolio.html > PORTFOLIO</a>
+  </nav>
+  </header>
+  <section class="hero" id="about">
+    <img 
+    src = "../assets/image/profilepic.JPG"
+    alt="Hang"
+    loading = "lazy"
+    class ="hero-img"
+    />
+    <div class="bio animate__animated animate__shakeX"></div>
+    <h2 class="bio-title">About Me</h2>
+      <p class="bio-text">
+          Natural science fanatic turned aspiring software developer after learning more about the art and science of computers and coding. While a seeker of new thrills and experiences, I enjoy a balance of routine at my hometown in Seattle. When I’m not at home as a kitchen chemist, you can find me exploring alpine lakes, mountains, and international lands. Wherever I am, I am dedicated to creating software, moving one step closer to my goal in creating products with less bias in the tech industry. 
+      </p>     
+</section>
+
+  <section>
+    <h2>Interest</h2>
+    <ul>
+      <li>
+        Pull-ups anywhere and everywhere
+      </li>
+      <li>
+        Connecting to my roots through cooking
+      </li>
+      <li>
+        Running. Ideally trails
+      </li>
+      <li>
+        Journaling
+      </li>
+      <li>
+        Diminishing food waste
+      </li>
+      <li>
+        Live music of all genres
+      </li>
+      <li>
+        Theater
+      </li>
+      <li>
+        Reading
+      </li>
+    </ul>
+
+    <h3 class = "likes">
+      <img 
+      src = "../assets/image/pullups.gif"
+      alt = "pull-ups"
+      width = 200
+      height = 350
+      />
+      <img 
+      src =  "../assets/image/music.jpg"
+      alt = "music"
+      width = 200
+      height = 350
+      />
+      <img 
+      src =  "../assets/image/cooking.jpeg"
+      alt = "cooking"
```

Comment:
It would be great if the alt text for the images were more descriptive. For someone with low vision/blindness, they depend on screen readers to tell them what the picture is. For example, this picture will just be called "cooking", which doesn't capture the beautiful dish you've created.